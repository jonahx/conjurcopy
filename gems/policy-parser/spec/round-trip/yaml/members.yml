- !group &developers developers

- !grant
  role: !group everyone
  members:
    - *developers
    - !group
      id: support
    - !group marketing
    - 
      role: !group ops
      admin: true
