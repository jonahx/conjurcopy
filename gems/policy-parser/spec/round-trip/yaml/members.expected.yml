---
- &1 !group
  id: developers
- !grant
  member:
  - !member
    role: *1
  - !member
    role: !group
      id: support
  - !member
    role: !group
      id: marketing
  - !member
    admin: true
    role: !group
      id: ops
  role: !group
    id: everyone
